tests/utils.py

```diff
@@ -391,6 +392,9 @@ def post_send_ex(agr_obj, send_object, gid_index, port, send_op=None, qp_idx=0):
         qp.wr_set_ud_addr(ah, agr_obj.rqps_num[qp_idx], agr_obj.UD_QKEY)
     if qp_type == e.IBV_QPT_XRC_SEND:
         qp.wr_set_xrc_srqn(agr_obj.remote_srqn)
+    if isinstance(agr_obj, Mlx5DcResources):
```

Comment:
What about old post_send? Or is DC only supported on extended QPs?

Reply:
It's only supported in extended QPs